wx/lib/agw/ultimatelistctrl.py

```diff
@@ -550,7 +549,7 @@ def __init__(self, parent):
 IL_VARIABLE_SIZE = 1
 
 # Python integers, to make long types to work with CreateListItem
-INTEGER_TYPES = six.integer_types
+INTEGER_TYPES = int
```

Comment:
This is changing the value from a list to a scalar value which is now causing crashes later on in the source. I converted all `in INTEGER_TYPES` to `== INTEGER_TYPES` and it fixed it. Should probably update the name to reflect the change as well.

`if type(itemOrId) in INTEGER_TYPES:` < line 598 

Another example:
```class CommandListEvent(wx.PyCommandEvent):
    """
    A list event holds information about events associated with :class:`UltimateListCtrl`
    objects.
    """

    def __init__(self, commandTypeOrEvent=None, winid=0):
        """
        Default class constructor.
        For internal use: do not call it in your code!

        :param `commandTypeOrEvent`: the event type or another instance of
         :class:`PyCommandEvent`;
        :param `winid`: the event identifier.
        """

        if type(commandTypeOrEvent) in INTEGER_TYPES:
````